invesalius/data/brainmesh_handler.py

```diff
@@ -7,32 +7,29 @@
 
 
 class Brain:
-    def __init__(self, img_path, mask_path, n_peels, affine_vtk):
+    def __init__(self, image, mask, n_peels, affine_vtk, ww, wl):
```

Comment:
Hmm, this is not related only to the code at hand, but I'm wondering more generally if it would benefit the project to start using the practice to write out variable names fully instead of abbreviating them. For example, here, it is hard to know what `ww` and `wl` refer to without a context. (A lot of existing code uses abbreviated variable names, but we could try to keep that principle with the new code added -- I think it would still increase the readability of the codebase in the long run.)

For reference, see, e.g.,

https://qvault.io/clean-code/naming-variables/ -- "8. Don’t use abbreviations or acronyms without sufficient context"

and

https://google.github.io/styleguide/cppguide.html -- "Minimize the use of abbreviations that would likely be unknown to someone outside your project (especially acronyms and initialisms). Do not abbreviate by deleting letters within a word. As a rule of thumb, an abbreviation is probably OK if it's listed in Wikipedia."

(That's Google's C++ style guide, but a similar principle probably applies to other languages, as well.)

In this particular instance, `ww` could be `window_width` and `wl` could be `window_length`.

I'm not sure how questions related to code style are decided with InVesalius, but maybe you can also consider this comment just in the context of this PR for now.

Reply:
you are correct. Avoiding abbreviations is a good idea. I'll use window_width and window_level

Reply:
Thanks a lot!